lib/uploadcare/resources/file.rb

```diff
@@ -95,6 +95,7 @@ def is_deleted?
       # copy file to target location
       # note what file copied with operations
       def copy with_operations=true, target=nil
+        warn "[DEPRECATION] `copy` is deprecated.  Please use `internal_copy` or `external_copy` instead."
```

Comment:
@dmitry-mukhin I've considered using Gem::Deprecate, but it produces messages like `NOTE: Uploadcare::Api::File#copy is deprecated; use #internal_copy or #external_copy instead. It will be removed on or after 2016-04-01` and there is no way to get rid of this removal date.

I've looked how ruby guys usually do deprecations and it seems that common practice is to either use `ActiveSupport::Deprecation` (for projects which depend on active_support, which is not our case) or use `warn` (example: [some random commit](https://github.com/rack/rack/commit/7c3d700c00d13e0dead8f110a95606fd1332c5a2) to Rack from one of the well-known ruby guys)